src/Illuminate/Cache/NullStore.php

```diff
@@ -25,6 +25,19 @@ public function get($key)
     }
 
     /**
+     * Retrieve multiple items from the cache by key.
+     *
+     * Items not found in the cache will have a null value for the key.
+     *
+     * @param  array  $keys
+     * @return array
+     */
+    public function getMultiple(array $keys)
+    {
+        //
```

Comment:
That makes sense.  I wasn't sure how to handle the null store.  I'll update that.